docs/current/quickstart/319191-quickstart-basics.mdx

```diff
@@ -12,7 +12,7 @@ import TabItem from "@theme/TabItem";
 
 ## Understand the basics of Dagger
 
-Before diving in to the code, here are some common questions about Dagger (and their answers).
+Before diving into the code, let us address some common questions about Dagger:
```

Comment:
```suggestion
Before diving into the code, let's address some common questions about Dagger:
```

Reply:
No, `let's` is correct IMO

Reply:
> No, `let's` is correct IMO

it is indeed. TIL!

Reply:
Solved in https://github.com/dagger/dagger/pull/4532/commits/5e8ec3722cdf68af6377b0329304076f9918b9a9